CondFormats/GeometryObjects/interface/HcalSimulationParameters.h

```diff
@@ -18,6 +18,14 @@ class HcalSimulationParameters {
   std::vector<int> pmtLeft_;
   std::vector<int> pmtFiberLeft_;
 
+  std::vector<int> hfLevels_;
```

Comment:
@bsunanda 
please make sure the evolution of  the CondFormats class  ( persistent type ) is limited to the pre* series. Once we have the 11_0_0, we won't be able to change the class any longer.